Assignment2/SandraTejedor.ttl

```diff
@@ -0,0 +1,14 @@
+@prefix class: <http://assignment2.org/class#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+class:Measurement8401 property:atTime "2010-06-12T12:00:12"^^<xsd:dateTime> ;

```

Comment:
It is xsd:dateTime, not <xsd:dateTime>. The same for the next line

Reply:
I can't see the error you are referring to

Reply:
Oh, sorry, in the comments the symbols "<" and ">" did not appear. I was referring to the fact that xsd:dateTime does not have to be included inside those symbols.